packaging/systemd/flotta.conf

```diff
@@ -1,4 +1,4 @@
 F /var/lib/systemd/linger/flotta 0755 root root
+d /var/home/flotta 0777 flotta flotta -
```

Comment:
If the order is important, can we document it?

Reply:
it isn't important. it is to ease the reading (I usually preferred sorted files).
from the [documentation](https://www.man7.org/linux/man-pages/man5/tmpfiles.d.5.html):

> When two lines are prefix path and suffix path of each other,
       then the prefix line is always created first, the suffix later
       (and if removal applies to the line, the order is reversed: the
       suffix is removed first, the prefix later)